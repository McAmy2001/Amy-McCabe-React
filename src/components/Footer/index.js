import React from 'react';

function Footer() {
  return (
    <footer>
      <h3>Find Me:</h3>
      <ul>
        <li>
          <a href="https://github.com/McAmy2001" target="_blank" rel="noopener noreferrer">GitHub</a>
        </li>
        <li>
          <a href="https://www.linkedin.com/in/amy-mccabe-mcamy2001" target="_blank" rel="noopener noreferrer">LinkedIn</a>
        </li>
        <li>
          <a href="https://www.facebook.com/amy.mccabe.754" target="_blank" rel="noopener noreferrer">Facebook</a>
        </li>
      </ul>
    </footer>
  )
}

export default Footer;