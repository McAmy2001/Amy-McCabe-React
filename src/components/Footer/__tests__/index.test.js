import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from '..';

afterEach(cleanup);

describe('Footer component', () => {
  it('renders', () => {
    render(<Footer></Footer>)
  });
  it('matches snapshot', () => {
    const { asFragment } = render(<Footer></Footer>);
    expect(asFragment()).toMatchSnapshot();
  });
})