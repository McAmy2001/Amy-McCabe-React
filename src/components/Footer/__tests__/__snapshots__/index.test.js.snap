// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer>
    <h3>
      Find Me:
    </h3>
    <ul>
      <li>
        <a
          href="https://github.com/McAmy2001"
          rel="noopener noreferrer"
          target="_blank"
        >
          GitHub
        </a>
      </li>
      <li>
        <a
          href="https://www.linkedin.com/in/amy-mccabe-mcamy2001"
          rel="noopener noreferrer"
          target="_blank"
        >
          LinkedIn
        </a>
      </li>
    </ul>
  </footer>
</DocumentFragment>
`;
